README.md

```diff
@@ -1,69 +1,128 @@
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
+### Key Features <a name="key-features"></a>
 
-## Built With
-- Ruby
-- irb
-- CMD with Ruby
-- rspec
-- sql
-- postgres
+- **Add comments**
+- **Like Posts**
 
-## Getting Started
+<p align="right">([back to top](#readme-top))</p>
 
-To get a copy of this project running on your local computer, do the followings:
+## 🚀 Live Demo <a name="live-demo"></a>
 
-### Prerequisites
-- Must have a working computer
-- A Web-Browser installed (Chrome - Recommended)
-- A code editor (VSCode - Recommended)
+- [N/A](https://yourdeployedapplicationlink.com)
 
-### Install
+<p align="right">([back to top](#readme-top))</p>
 
-Follow all steps to install [_Ruby_](https://rubyinstaller.org/) 
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Setup
+Clone project
 
-- Open the terminal on your computer
-- Navigate through directories using [change directory (cd)](https://www.howtogeek.com/659411/how-to-change-directories-in-command-prompt-on-windows-10) command to where you want to clone the repository.
+```bash
+git clone https://github.com/Mov305/blog-rails.git
+```
 
-- **Type the following lists of commands and press "ENTER" after each command:**
+cd blog-app-rails
+```
 
-- **Open folder in VSC**
+Run the program
 
-  - code .
+```bash
+rails s
+```
 
-  - _Install dependencies locally_
-    - bundle install
-    - then run the command `bundle install` to install all the dependencies
-  - _Run the application_
-    - run the command `rails s` to run the application
+<p align="right">([back to top](#readme-top))</p>
 
 
-## Authors
+## 👥 Authors <a name="authors"></a>
 
 👤 **Abdelrhman Samy Saad**
 
 - GitHub: [@Mov305](https://github.com/Mov305)
 - Twitter: [@Mov305](https://twitter.com/Mov_abd)
 - LinkedIn: [@Mov305](https://www.linkedin.com/in/abdelrhman-samy-80b14b215/)
 
+<p align="right">([back to top](#readme-top))</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **More methods**
+
+<p align="right">([back to top](#readme-top))</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/paulinagonzalezc/blog-app-rails/issues).
+
+<p align="right">([back to top](#readme-top))</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star 😁🌟✨
+
+<p align="right">([back to top](#readme-top))</p>
 
-## Acknowledgements ⭐️
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-- Special thanks to the team for being fully commited on finishing this project together.
+Microverse.
 
-## Show your support
+<p align="right">([back to top](#readme-top))</p>
 
-- Give a ⭐️ if you like this project!
+## 📝 License <a name="license"></a>
```

Comment:
Great job @Mov305 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.